code/game/objects/items/surgery_tray.dm

```diff
@@ -0,0 +1,126 @@
+/datum/storage/medicart/New()
+	.=..()
+	max_total_storage = 30
+	max_specific_storage = WEIGHT_CLASS_NORMAL
+	max_slots = 14
+	set_holdable(list(
+				/obj/item/hemostat,
+				/obj/item/scalpel,
+				/obj/item/retractor,
+				/obj/item/circular_saw,
+				/obj/item/surgicaldrill,
+				/obj/item/cautery,
+				/obj/item/bonesetter,
+				/obj/item/surgical_drapes,
+				/obj/item/clothing/mask/surgical,
+				/obj/item/razor,
+				/obj/item/blood_filter,
+				/obj/item/stack/sticky_tape/surgical,
+				/obj/item/stack/medical/bone_gel,
+	))
```

Comment:
```suggestion
	set_holdable(list(
		/obj/item/hemostat,
		/obj/item/scalpel,
		/obj/item/retractor,
		/obj/item/circular_saw,
		/obj/item/surgicaldrill,
		/obj/item/cautery,
		/obj/item/bonesetter,
		/obj/item/surgical_drapes,
		/obj/item/clothing/mask/surgical,
		/obj/item/razor,
		/obj/item/blood_filter,
		/obj/item/stack/sticky_tape/surgical,
		/obj/item/stack/medical/bone_gel,
	))
```
Also, would be cool to alphabetize this list. If you got an addon which does it for you it'd be pretty easy.

Reply:
done, thanks for the tip 👍